src/tracker.js

```diff
@@ -0,0 +1,3 @@
+/* @flow */
+
+export { Tracker } from './tracker-component';
```

Comment:
how can one be sure Tracker object is available on the global?

Reply:
it's magic. when components get pulled into the sdk, any exports are automatically attached to window.paypal

Reply:
👍